src/arrayMethodReduce.js

```diff
@@ -5,7 +5,20 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let sum = initialValue;
```

Comment:
It does not always sum:
```suggestion
    let acumulator = initialValue;
```